packages/store/src/internal/state-factory.ts

```diff
@@ -155,14 +154,14 @@ export class StateFactory {
    * Invoke the init function on the states.
    */
   invokeInit(stateMetadatas: MappedStore[]) {
```

Comment:
```ts
export LifeCycleStateManager {

}
```

0 - rename ngxsBootstrap to invokeNgxsBootstrap
1 - create file: lifecycle-state-manager.ts
2 - move invokeNgxsBootstrap, invokeInit, invokeAfterBootstrap to lifecycle-state-manager.ts